.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo-token: 52OLLNCnPFH2EgHdDBFz19cOGsLAmmhri
```

Comment:
coveralls does not require repo token on travis. You can add this file to .gitignore.

Reply:
We can't have the repo token on public repository for security reasons. Any one with that token can upload code coverage to our codecov project.